WORKSPACE

```diff
@@ -117,44 +69,17 @@ com_github_grpc_grpc_bazel_grpc_deps()
 load("@stackb_rules_proto//java:deps.bzl", "java_grpc_compile")
 java_grpc_compile()
 
-# Load GRPC Python dependencies
-load("@stackb_rules_proto//python:deps.bzl", "python_grpc_compile")
-python_grpc_compile()
-
-# Load GRPC Node.js dependencies
-load("@stackb_rules_proto//node:deps.bzl", "node_grpc_compile")
-node_grpc_compile()
-
 
 ########################################
 #     Load Deployment Dependencies     #
 ########################################
 
-git_repository(
-    name="graknlabs_bazel_distribution",
-    remote="https://github.com/graknlabs/bazel-distribution",
-    commit="796cea2531404509a3298af65ab562b1929c3eb6"
-)
+load("//dependencies/distribution:dependencies.bzl", "distribution_dependencies")
+distribution_dependencies()
 
 load("@graknlabs_bazel_distribution//github:dependencies.bzl", "github_dependencies_for_deployment")
```

Comment:
Why is `@graknlabs_bazel_distribution//github:dependencies.bzl` not included in `@graknlabs_bazel_distribution//distribution:dependencies.bzl`? I.e. why do we need to load 2 macros from `@graknlabs_bazel_distribution`?

Reply:
`distribution_dependencies` is the macro which loads `git_repository(graknlabs_bazel_distribution)`. Then you need to load individual functionalities within it, which is `github_dependencies_for_deployment` in this particular example.

This is similar to how we do it in the other sections, eg., GRPC:
```
# Load GRPC dependencies
load("//dependencies/compilers:dependencies.bzl", "grpc_dependencies")
grpc_dependencies()

load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", com_github_grpc_grpc_bazel_grpc_deps = "grpc_deps")
com_github_grpc_grpc_bazel_grpc_deps()

# Load GRPC Java dependencies
load("@stackb_rules_proto//java:deps.bzl", "java_grpc_compile")
java_grpc_compile()
```

Reply:
ah i see. We can't load `@graknlabs_bazel_distribution//github:dependencies.bzl` in `//dependencies/distribution:dependencies.bzl` I suppose then? If not then yeah we gotta keep it.

Reply:
Cool. Yeah I don't see any other way